docker-compose.env.yaml.example

```diff
@@ -0,0 +1,4 @@
+DEBUG=1
```

Comment:
Is the `.yaml` part in the filename on purpose?

Reply:
Yeah, the actual file is called `docker-compose.env.yaml` and this is an example of that.